scripts/release/release.go

```diff
@@ -4,29 +4,22 @@ import (
 	"bytes"
 	"fmt"
 	"os"
-	"regexp"
-	"strings"
 
-	"github.com/blang/semver/v4"
+	"github.com/cri-o/cri-o/internal/version"
+	"github.com/cri-o/cri-o/scripts/utils"
 	goGit "github.com/go-git/go-git/v5"
 	"github.com/sirupsen/logrus"
 	"sigs.k8s.io/release-sdk/git"
 	"sigs.k8s.io/release-sdk/github"
 	"sigs.k8s.io/release-utils/env"
-	"sigs.k8s.io/release-utils/util"
 )
 
 const (
-	githubTokenEnvKey = "GITHUB_TOKEN"
-	gitRemoteEnvKey   = "REMOTE"
-	orgEnvKey         = "ORG"
-	versionFile       = "internal/version/version.go"
-	branchPrefix      = "release-"
-	crioOrgRepo       = "cri-o"
+	versionFile       = utils.VersionFile
+	branchPrefix      = utils.BranchPrefix
+	githubTokenEnvKey = utils.GithubTokenEnvKey
```

Comment:
Let's use them directly in the script and not re-define them. Like instead of `branchPrefix` we can directly use `utils.BranchPrefix` here.